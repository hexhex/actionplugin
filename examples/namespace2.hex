#namespace("ex","http://example.org#").
#namespace("","http://www.kr.tuwien.ac.at/default/").
a("ex:foo").
b("ex:foo").
d("http://example.org#bar").
d("http://www.kr.tuwien.ac.at/default/bar").
d("bar").
d(":bar").
d("foo:bar").
c(X) :- a(X), b(X).
