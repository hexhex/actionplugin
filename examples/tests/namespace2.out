{c("<http://example.org#foo>"), d("foo:bar"), d("<http://www.kr.tuwien.ac.at/default/bar>"), d("bar"), d("http://www.kr.tuwien.ac.at/default/bar"), d("http://example.org#bar"), b("<http://example.org#foo>"), a("<http://example.org#foo>")}
